<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClockForm</class>
 <widget class="QWidget" name="ClockForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clock</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>79</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QLabel" name="label_4">
      <property name="maximumSize">
       <size>
        <width>4</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="secondsLbl">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>00</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="hoursLbl">
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="minutesLbl">
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>00</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>4</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>37</y>
     <width>741</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Clock" name="clockWgt" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Clock</class>
   <extends>QWidget</extends>
   <header>clock_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
